Documentation/common_server_docs.py

```diff
@@ -147,6 +147,9 @@ def create_py_documentation(path, origin, language):
     with open(path, 'r') as file:
         py_script = YmlUnifier.clean_python_code(file.read(), remove_print_future=False)
 
+    logging.info("replacing DemistoClassApiModule: ")
+    py_script = re.sub(r'from DemistoClassApiModule import \*[ \t]*(#.*)?', "", py_script)
```

Comment:
why is this required?

Reply:
We cant import it if we want to run CommonServerPython module, it fails...